File: aoc2023/test/aoc2023/day05_test.clj
(ns aoc2023.day05-test
  (:require [clojure.test :refer :all]
            [clojure.java.io]
            [aoc2023.day05 :as day05]))

(def sample-input (slurp (clojure.java.io/resource "day05-sample.txt")))

(deftest test-part1
  (testing "Day 5 Part 1"
    (is (= 35 (day05/part1 sample-input)))
  )
)

;; (deftest test-part2
;;   (testing "Day 5 Part 2"
;;     (is (= 30 (day05/part2 sample-input)))
;;   )
;; )

File: 2024/day05/day05_test.go
package day05

import (
	"fmt"
    "testing"
    )

var sample = "sample.txt"

func TestPart1(t *testing.T) {
  result := Part1(sample)
  if result != 143 {
    fmt.Println(result)
  	t.Fail()
  }
}

func TestPart2(t *testing.T) {
  result := Part2(sample)
  if result != 123 {
    fmt.Println(result)
  	t.Fail()
  }
}
